meson/meson-dist-script

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eu
+
+cd "${MESON_DIST_ROOT}"
+
+# Delete everything we do not want to have in the release tarballs:
+rm -rf \
+   .clang-format \
+   .editorconfig \
+   travis \
+   .github
+
+mkdir build
+cd build
+meson .. -Dprefix=/usr
+ninja
+cd ..
+rm -rf build
```

Comment:
As for this... what is the purpose of this? You're building it but not doing anything with the results, not even copying files out?

Reply:
Yeah, that was copied from i3. Removing in https://github.com/i3/i3lock/pull/320